starter-code/main.js

```diff
@@ -57,6 +59,227 @@ mongoClient.connect(url, (error, db) => {
+                  }
+                })
+            break;
+          
+              case "6":
+                //What companies have offices in "Barcelona".
+                db.collection('companies').find({'offices.city' : 'Barcelona' }, {name: 1, _id: 0}).toArray((error, result) => {
+                    if (error) {
+                      console.log(error);
+                      rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                    } else {
+                      console.log(result);
+                      rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                    }
+                  })
+            break;
+          
+            case "7":
+              //List the 10 companies with more employees sorted ascending (show name and employees).
+              db.collection('companies').find({}, {name: 1, _id: 0}).sort( {'number_of_employees': -1} ).limit(10).toArray((error, result) => {
+                if (error) {
+                  console.log(error);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                } else {
+                  console.log(result);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                }
+              })
+            break;
+          
+            case "8":
+              //Find the company with the name "Facebook".
+              db.collection('companies').find({'name': 'Facebook'}, {name: 1, _id: 0}).toArray((error, result) => {
+                if (error) {
+                  console.log(error);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                } else {
+                  console.log(result);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                }
+              })
+            break;
+          
+            case "9":
+              //How many employees has Facebook?.
+              db.collection('companies').find({'name': 'Facebook'}, {name: 1, number_of_employees: 1, _id: 0}).toArray((error, result) => {
+                if (error) {
+                  console.log(error);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                } else {
+                  console.log(result);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                }
+              })
+            break;
+          
+            case "10":
+              //List the name of all the products of Facebook
+            db.collection('companies').find({ 'name': 'Facebook' }, { name: 1, 'products.name': 1, _id: 0 }).toArray((error, result) => {
+                if (error) {
+                  console.log(error);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                } else {
+                  console.log(result);
```

Comment:
```suggestion
                  console.log(result[0]);
```
De esta manera puedes acceder 😃